README.md

```diff
@@ -70,9 +70,9 @@ and import it into File→Settings→Editor→Code Style.
 
 ### Eclipse
 
-Version 1.6 of the
-[google-java-format Eclipse plugin](https://github.com/google/google-java-format/releases/download/google-java-format-1.6/google-java-format-eclipse-plugin_1.6.0.jar)
-can be downloaded from the releases page. Drop it into the Eclipse
+The current version of the google-java-format Eclipse plugin can be downloaded
+from the [releases page](https://github.com/google/google-java-format/releases)
+(`google-java-format-eclipse-plugin-X.jar`). Drop the jar into the Eclipse
```

Comment:
Thanks - these docs have already been updated in f2eabaa9a7a2fe1ccd17c30124d2041fe97a6313